misc/config_tools/scenario_config/config_summary.py

```diff
@@ -302,7 +302,10 @@ def get_basic_information_table(self, vm_node):
                 vcpu_num.add(item)
         amount_vm_l3_cache = self.get_amount_l3_cache(vm_node)
         parameter_dict["Load Order"] = load_order
-        parameter_dict["Number of vCPUs"] = len(vcpu_num)
```

Comment:
Since `vcpu_num` is no longer used, remove line 281 and line 297-302 as well.

Reply:
Done